client/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script defer src=""></script>
+
+    <title>Anonymous Entries</title>
+</head>
+<body>
+
+    <h1 class="blogTitle">Anonymous Entries</h1>
+
+    
+    <form>
```

Comment:
Front end looks lovely! You could add autocomplete = 'off' to your form. This removes the autofill in your form to make it more anonymous